implementation/src/main/java/io/smallrye/metrics/MetricsRequestHandler.java

```diff
@@ -102,9 +105,10 @@ public void handleRequest(String requestPath,
         }
 
         if (!requestPath.startsWith(contextRoot)) {
-            responder.respondWith(500, "The expected context root of metrics is "
-                    + contextRoot + ", but a request with a different path was routed to MetricsRequestHandler",
-                    Collections.emptyMap());
+            String message = "The expected context root of metrics is "
+                    + contextRoot + ", but a request with a different path was routed to MetricsRequestHandler";
```

Comment:
Just for convenience I would use String.format but this is in this case just my preference.

Reply:
I guess it doesn't really matter much, but in other places in the codebase we mostly use + concatenation, so I'd keep it consistent

Reply:
Here it does not matter. But if you have some intense logging than it could have some performance impact. But apart from micro benchmarks I cannot say how severe this impact is...